test/functional/tests/stats/test_ioclass_stats.py

```diff
@@ -9,328 +9,666 @@
+        cache.set_seq_cutoff_policy(SeqCutOffPolicy.never)
+
+    with TestRun.step(f"Add core to cache"):
+        core = cache.add_core(core_dev=core_device)
+
+    with TestRun.step("Reset counter for each core"):
+        core.reset_counters()
+
+    with TestRun.step("Create mountpoint directory"):
+        create_directory(path=mountpoint)
+
+    with TestRun.step("Prepare IO class config file and save it"):
+        ioclass_list = [
+            IoClass(
+                class_id=class_id,
+                rule=f"file_size:le:{4096 * class_id}&done",
+                priority=22,
+            )
+            for class_id in range(min_ioclass_id, max_ioclass_id)
+        ]
+        IoClass.save_list_to_config_file(ioclass_list, True)
 
-    with TestRun.step("Test prepare"):
-        prepare(random_cls)
+    with TestRun.step("Load IO class config file"):
+        cache.load_io_class(ioclass_config.default_config_file_path)
 
-    with TestRun.step("Prepare IO class config file"):
-        ioclass_list = []
+    with TestRun.step("Generate files with particular sizes in temporary folder"):
+        file_list = []
         for class_id in range(min_ioclass_id, max_ioclass_id):
-            ioclass_list.append(IoClass(
+            path = f"/tmp/open_cas_test_data/test_file_{file_size_base * class_id}"
+            File.create_file(path)
+            file = File(path)
+            file.padding(Size(file_size_base * class_id, Unit.Byte))
+            file_list.append(file)
+
+    with TestRun.step("Mount core"):
+        core.mount(mountpoint)
+
+    with TestRun.step("Prepare IO class config file and save it"):
+        ioclass_list = [
+            IoClass(
                 class_id=class_id,
                 rule=f"file_size:le:{4096 * class_id}&done",
-                priority=22
-            ))
+                priority=22,
+            )
+            for class_id in range(min_ioclass_id, max_ioclass_id)
+        ]
         IoClass.save_list_to_config_file(ioclass_list, True)
 
     with TestRun.step("Load IO class config file"):
         casadm.load_io_classes(cache_id, file=ioclass_config.default_config_file_path)
 
-    with TestRun.step("Try retrieving IO class stats for all allowed id values "
-                      "and one out of range id"):
+    with TestRun.step(
+        "Try retrieving IO class stats for allowed ids values and one out of range id"
+    ):
         for class_id in range(ioclass_config.MAX_IO_CLASS_ID + 2):
-            out_of_range = " out of range" if class_id > ioclass_config.MAX_IO_CLASS_ID else ""
-            with TestRun.group(f"Checking{out_of_range} IO class id {class_id}..."):
-                expected = class_id == 0 or class_id in range(min_ioclass_id, max_ioclass_id)
+            out_of_range = (
+                " out of range" if class_id > ioclass_config.MAX_IO_CLASS_ID else ""
+            )
+
+            with TestRun.group(f"Checking {out_of_range} IO class id {class_id}..."):
```

Comment:
```suggestion
            with TestRun.group(f"Checking{out_of_range} IO class id {class_id}..."):
```
To avoid double space in log